bin/smoke-test.sh

```diff
@@ -11,7 +11,12 @@ for TRANSFORM in $(ls ./tests/fixtures); do
     mkdir -p "/tmp/lwc-codemod-tests/${TRANSFORM}"
     cp -R "./tests/fixtures/${TRANSFORM}/${DIR}" "/tmp/lwc-codemod-tests/${TRANSFORM}/${DIR}"
 
-    node ./transforms/cli.js "${TRANSFORM}" "/tmp/lwc-codemod-tests/${TRANSFORM}/${DIR}"
+    CLI_PATH="$(pwd)/transforms/cli.js"
+
+    # intentionally test with a relative file path since that's what most users will do
+    cd /tmp/lwc-codemod-tests
+    node "$CLI_PATH" "${TRANSFORM}" "${TRANSFORM}/${DIR}"
+    cd -
```

Comment:
This is not strictly necessary for the repro, but I feel like it's a good safety measure. Most users will do `npx lwc-codemod <mod> ./path/to/something` rather than provide a full absolute path.